<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MST-101-HP" name="Firefox Test" tests="3" failures="1" timestamp="1 Feb 2017 10:13:25 GMT" time="209.323" errors="0">
  <testcase name="leadCreation" time="39.16" classname="main_page.TestEngine"/>
  <testcase name="accountCreation" time="65.382" classname="main_page.TestEngine"/>
  <testcase name="contactCreation" time="61.805" classname="main_page.TestEngine">
    <failure type="exception.CustomException" message="The value TestAutomationis not matched with the given value Test">
      <![CDATA[exception.CustomException: The value TestAutomationis not matched with the given value Test
	at Page_Object_page.contactPage.accountNameVerify(contactPage.java:88)
	at main_page.TestEngine.contactCreation(TestEngine.java:149)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- contactCreation -->
</testsuite> <!-- Firefox Test -->
